src/utils/craftLayerURL.js

```diff
@@ -10,8 +12,15 @@ export const createLayerURL = (location, type) => {
     pathname,
     search,
   } = location;
+  const splittedPath = pathname.split('/');
 
-  const url = `${pathname}${search}`;
+  if (type === LAYER_TYPES.CREATE) {
+    splittedPath.length = 4;
```

Comment:
This shouldn't be necessary. `.push()` increases the size of the array automatically. See: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push

Reply:
Hi @ryandberger! 
I used `splittedPath.length = 4` to **cut first four elements from the path**. Because we can get path like this `/settings/data-import/profileType?sort=name` or `/settings/data-import/profileType/view/profileId?sort=name`.
And as the result we get array like this `['', 'settings', 'data-import', 'profileType']`.
Then I push the layer type `create` and join this array. As the result we get path `/settings/data-import/profileType/create?sort=name`